### ISSUE TITLE
Back up critical repos

### ISSUE THREAD

**Issue by nmcain** [Horus125/fuchsia-ui#1]
the repos in the script could be deleted at any time and should be bankedup

**Comment by Horus125** [comment]
Good point. I have already forked them, if the official repos get deleted and links in the script stop working I'll change them to my copies on github
